### DISCUSSION TITLE
代理地址https://chat.duti.tech/什么用？可以自己搭建吗？

### DISCUSSION THREAD

**Discussion by MillionWang0724** [acheong08/ChatGPT#784] [Q&A]
代理地址https://chat.duti.tech/什么用？可以自己搭建吗？

**Comment by acheong08** [comment]
https://github.com/acheong08/ChatGPT-Proxy-V1

问题是`config.json` 不在因为OpenAI很容易把漏洞给补上

**Reply by PenberRay** [reply]
请问这个具体怎么使用呢

**Reply by jianhai0527** [reply]
同问。。这个project不存在了

**Comment by MillionWang0724** [comment]
感谢，config.json什么的没明白



---- 回复的原邮件 ----
| 发件人 | Antonio ***@***.***> |
| 日期 | 2023年02月16日 18:58 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [acheong08/ChatGPT] 代理地址https://chat.duti.tech/什么用？可以自己搭建吗？ (Discussion #784) |

https://github.com/acheong08/ChatGPT-Proxy-V1

问题是config.json 不在因为OpenAI很容易把漏洞给补上

**Reply by acheong08** [reply]
The configuration defines the endpoint which leaks the API without cloudflare. It is hidden so that OpenAi does not stop it

**Comment by lozn00** [comment]
chat.duti.tech 有啥作用,v2版本基本上都是使用的chat.duti.tech ,只是换了个ip地址么,目前是404
